wireguard-server/install-wgrest.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+install_wgrest() {
+  ARCH=""
+  case $(uname -m) in
+  x86_64)
+    ARCH="amd64"
+    ;;
+  aarch64)
+    ARCH="arm64"
+    ;;
+  **)
+    echo "ERROR: The system architecture isn't supported"
+    exit 1
+    ;;
+  esac
+  curl -L https://github.com/suquant/wgrest/releases/latest/download/wgrest_$ARCH.deb -o wgrest_amd.deb
+  dpkg -i wgrest_amd.deb
```

Comment:
We should pin the version when downloading the dependency

We should also ask curl to throw an error when there's an HTTP error (eg 500, 404).
This ensures we get an error instead of writing gibberish to the file.
```
  wgrest_version=1.0.0-alpha10.5
  # ...
  curl -fssL https://github.com/suquant/wgrest/releases/download/${wgrest_version}/wgrest_$ARCH.deb -O
  dpkg -i wgrest_$ARCH.deb
```

Reply:
Fixed - thanks!